pkgs/by-name/be/beeper/package.nix

```diff
@@ -15,19 +15,27 @@ let
     hash = "sha256-K043RQ5BoS1ysnmY+LpRixBmMx2XCbRzhWnWsxg26dg=";
   };
   appimageContents = appimageTools.extract {
-    inherit version pname src;
+    inherit pname version src;
+
+    postExtract = ''
+      # disable creating a desktop file and icon in the home folder during runtime
+      linuxConfigFilename=$out/resources/app/build/main/linux-*.mjs
+      echo "export function registerLinuxConfig() {}" > $linuxConfigFilename
+
+      # disable auto update
+      sed -i 's/[^=]*\.auto_update_disabled/true/' $out/resources/app/build/main/main-entry-*.mjs
+
+      # prevent updates
+      sed -i -E 's/executeDownload\([^)]+\)\{/executeDownload(){return;/g' $out/resources/app/build/main/main-entry-*.mjs
+    '';
   };
 in
-appimageTools.wrapType2 {
-  inherit pname version src;
+appimageTools.wrapAppImage {
```

Comment:
Due to this package needing `nix-update`, we won't be able to use `wrapAppImage`

Reply:
Another option would be to avoid using nix-update in the update script, since it's not that difficult to replace version and hash by other means. See for example https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/tr/trilium-next-desktop/update.sh